src/main/java/org/stellar/sdk/Transaction.java

```diff
@@ -258,6 +267,7 @@ public String toEnvelopeXdrBase64() {
     private TimeBounds mTimeBounds;
     List<Operation> mOperations;
     private boolean timeoutSet;
+    private int baseFee;
```

Comment:
Same thought as above (and this should apply in other place too). remove `base` so it's just called `fee`.

Reply:
We should actually change it to something like `perOperationFee` as it's multiplied by number of operations below.

Reply:
ok, will use perOperationFee so it's more descriptive.